code/__DEFINES/say.dm

```diff
@@ -17,6 +17,7 @@
 #define MODE_HOLOPAD "holopad"
 #define MODE_CHANGELING "changeling"
 #define MODE_VOCALCORDS "cords"
+#define MODE_MONKEY "monkeyhive"
```

Comment:
Nope we're not adding more of say modes.

Reply:
Why not?

Reply:
i'm not adding a snowflake verb

Reply:
@AnturK yeah why not?

Reply:
The way they are coded ever single one adds significant overhead on every single say, handling listeners/senders is done very ambiguously and the system is the kind of ancient mess that needs proper rewrite instead of piling more onto it. Especially for what is rehash of mode that already exists in five variations.

Reply:
... saymode datums? Is checking to see if ```SSradio.saymodes["monkeyhive"]``` exists really THAT much of an overhead?